admin/client/App/screens/Item/index.js

```diff
@@ -75,7 +75,7 @@ var ItemView = React.createClass({
 	renderRelationships () {
 		const { relationships } = this.props.currentList;
 		const keys = Object.keys(relationships);
-		if (!keys.length) return;
+		if (!keys.length) return null;
 		return (
```

Comment:
without the hard return null a load of a search query URI with zero results was getting a react render error